### ISSUE TITLE
[BUG] `pyperclip` is not listed as a dependency

### ISSUE THREAD

**Issue by ThatIsAPseudo** [Sxvxgee/UnlimitedGPT#35]
### Description

Importing the UnlimitedGPT library will result in a `ModuleNotFoundError: No module named 'pyperclip'`

### Steps to Reproduce

Run the following code.

### Code used
Please provide the code that you were using when you encountered the bug.

```python
from UnlimitedGPT import ChatGPT
```

### Error Message or Traceback

If applicable, include the complete error message or traceback you received. This will help us in identifying the problem more quickly.

```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
[...]/chatgpt.py in line 1
----> 1 from UnlimitedGPT import ChatGPT

File [...]/python3.11/site-packages]/UnlimitedGPT/__init__.py:7
      1 """
      2 ChatGPT.py
      3 
      4 An unofficial Python wrapper for OpenAI's ChatGPT API
      5 """
----> 7 from UnlimitedGPT.UnlimitedGPT import ChatGPT

File [...]/python3.11/site-packages/UnlimitedGPT/UnlimitedGPT.py:3
      1 import datetime
      2 import re
----> 3 import pyperclip
      4 from json import loads
      5 from logging import DEBUG, Formatter, StreamHandler, getLogger

ModuleNotFoundError: No module named 'pyperclip'
```

### Environment Information

Please provide details about your environment:

- Python version: 3.11
- Operating System: macOS Monterey
- UnlimitedGPT version: 0.1.9.2

### Expected Behavior

All dependencies should be listed in the `requirements.txt`file, and installed along UnlimitedGPT

### Possible Solutions

Add `pyperclip` to `requirements.txt`

### Checklist

Before submitting the issue, please ensure that you have done the following:

- [x] I have reviewed the documentation before submitting this but report.
- [x] I have confirmed that I am running the latest version of UnlimitedGPT.
- [x] I have verified that the issue is not caused by any modifications I made to the library or code.
- [x] I have confirmed that the issue is caused by the library itself and not other dependencies used. [E.G headless mode not working because of undetected-chromedriver]

**Comment by Sxvxgee** [comment]
Hi, thanks for making this issue, and an extra thanks for following the bug template. I'll be releasing a new version to the library soon with a couple fixes that include adding pyperclip to the requirements.txt file. In the meantime, if you haven't already, just install pyperclip as a normal library and that should fix the issue

**Comment by Sxvxgee** [comment]
Fixed in `0.1.9.3`. Please update to the new version to fix this issue. Thanks for reporting the bug!
